http/swagger.yml

```diff
@@ -4341,6 +4341,11 @@ paths:
           schema:
             type: string
           description: Filter tasks to a specific organization ID.
+        - in: query
+          name: status
+          schema:
+            type: string
+          description: Filter tasks by a status--"inactive" or "active".
```

Comment:
Great! Can you add an `enum` annotation to list the only valid values as `active` and `inactive`, similar to:

https://github.com/influxdata/influxdb/blob/1a28a3256e50b44afb547a14b997c2b0cf7c4527/http/swagger.yml#L6698-L6701

This will improve documentation and code generation.